src/modules/account/components/AddAccountBySecretRecovery/AddAccountBySecretRecovery.js

```diff
@@ -6,26 +6,28 @@ import SetPasswordForm from 'src/modules/auth/components/SetPasswordForm/SetPass
 import MultiStep from 'src/modules/common/components/MultiStep';
 import SetPasswordSuccess from 'src/modules/auth/components/SetPasswordSuccess';
 import routes from '@views/screens/router/routes';
-import { useCurrentAccount } from '@account/hooks';
+import { useCurrentAccount, useAccounts } from '@account/hooks';
 import AddAccountForm from '../AddAccountForm';
 import styles from './AddAccountBySecretRecovery.css';
 
 const AddAccountBySecretRecovery = ({ history, login }) => {
   const multiStepRef = useRef(null);
   const [passphrase, setPassphrase] = useState(null);
   const [currentAccount, setCurrentAccount] = useCurrentAccount();
+  const { setAccount } = useAccounts();
   const onAddAccount = (passphraseData) => {
     setPassphrase(passphraseData);
     multiStepRef.current.next();
   };
 
   const onSetPassword = (account) => {
     setCurrentAccount(account);
+    setAccount(account);
     multiStepRef.current.next();
   };
 
   const onPasswordSetComplete = () => {
-    login(passphrase); // Todo this login method is depricated
+    login(passphrase); // Todo this login method is deprecated
```

Comment:
can we not use login, signin or any other terminology to avoid further confusion? if you don't the terminology we can brainstorm with the team?

following are some initial thoughts:
- authenticate? : when a user choose an account and navigates to wallet
- authorize?: when a user choose to use his wallet (any action associated with token usage, whereby requiring password)